docs/src/pages/theming.mdx

```diff
@@ -242,6 +242,7 @@ component match your design in the exact way you want.
 ```
 
 <UploadButton
+  endpoint="mockRoute"
```

Comment:
having these actually load instead of fetch failing helped reduce some impossible states we had weird paths for just to display in here